modules/eks/README.md

```diff
@@ -20,22 +20,24 @@
-| eks\_map\_accounts            | Additional AWS account numbers to add to the aws-auth configmap                                                                                        | n/a                  |   yes    |
-| eks\_map\_roles               | Additional IAM roles to add to the aws-auth configmap                                                                                                  | n/a                  |   yes    |
-| eks\_map\_users               | Additional IAM users to add to the aws-auth configmap                                                                                                  | n/a                  |   yes    |
-| network                       | Network where the Kubernetes cluster will be hosted                                                                                                    | n/a                  |   yes    |
-| node\_pools                   | An object list defining node pools configurations                                                                                                      | `[]`                 |    no    |
-| node\_pools\_launch\_kind     | Which kind of node pools to create. Valid values are: launch\_templates, launch\_configurations, both.                                                 | `"launch_templates"` |    no    |
-| resource\_group\_name         | Resource group name where every resource will be placed. Required only in AKS installer (*)                                                            | `""`                 |    no    |
-| ssh\_public\_key              | Cluster administrator public ssh key. Used to access cluster nodes with the operator\_ssh\_user                                                        | n/a                  |   yes    |
-| subnetworks                   | List of subnets where the cluster will be hosted                                                                                                       | n/a                  |   yes    |
-| tags                          | The tags to apply to all resources                                                                                                                     | `{}`                 |    no    |
+| Name                                     | Description                                                                                                                                            | Default              | Required |
+|------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|:--------:|
+| cluster\_endpoint\_public\_access        | Indicates whether or not the Amazon EKS public API server endpoint is enabled.                                                                         | `false`              |    no    |
```

Comment:
If this is a boolean parameter, IMO is better to add `enabled` to the name so it is obvious what happens when you set it to true/false.
```suggestion
| cluster\_endpoint\_public\_access\_enabled        | Indicates whether or not the Amazon EKS public API server endpoint is enabled.                                                                         | `false`              |    no    |
```

Reply:
@ralgozino we took it verbatim from the aws' module to avoid getting lost in translation

Reply:
in that case, I can live with that :)